libs/datamodel/connectors/dml/src/native_type_constructor.rs

```diff
@@ -45,3 +46,43 @@ impl NativeTypeConstructor {
         }
     }
 }
+
+impl From<MsSqlType> for NativeTypeConstructor {
```

Comment:
The conversion from `MssqlType` to `NativeTypeConstructor` does not belong in this crate. This is only the interface and must not contain implementation details.

Reply:
If we add it into `native-types` crate, we'll create a circular dependency with the `dml`.

Reply:
We shouldn't use `From` here then.